cmd/crio/main.go

```diff
@@ -197,9 +197,9 @@ func main() {
 			Name:  "conmon",
 			Usage: "path to the conmon executable",
 		},
-		cli.BoolFlag{
-			Name:  "debug",
-			Usage: "enable debug output for logging",
+		cli.StringFlag{
+			Name:  "log-level",
+			Usage: "log CRI-O messages above specified level: debug, warn, error, fatal or panic",
```

Comment:
log CRI-O messages above specified level: debug, warn(default), error, fatal or panic",

Reply:
FWIW, the (default) add puts us 5 characters over 80.....